src/dodal/common/data.py

```diff
@@ -0,0 +1,145 @@
+from abc import ABC, abstractmethod
+from pathlib import Path
+from typing import Optional
+
+from aiohttp import ClientSession
+from ophyd_async.core import DirectoryInfo, DirectoryProvider
+from pydantic import BaseModel
+
+
+class DataCollectionIdentifier(BaseModel):
+    # VisitService API
```

Comment:
?

Reply:
Just marking that these classes are part of the partial VisitService implemenation in the GDA server. I can remove.

Reply:
Happy for you to keep it if you link to the overarching JIRA/GH issue so that people like me can figure out what "VisitService API" means ;) It's just not clear to an outsider currently what is meant to be permanent and what is a stop-gap solution.